tsconfig.json

```diff
@@ -1,39 +1,16 @@
 {
-  "compileOnSave": true,
   "compilerOptions": {
     "target": "ES2020",
     "module": "commonjs",
-    "allowJs": false,
-    "jsx": "preserve",
-    "outDir": "./dist",
-    "rootDir": ".",
-    "baseUrl": ".",
-    "paths": {
-      "*": ["src/*", "*"]
-    },
-    "isolatedModules": true,
+    "lib": ["es2015", "es2016", "es2017"],
```

Comment:
Are these the same settings we have in our other typescript repos? I'd like to be consistent across all our repos.

Reply:
I copied these from market-maker-js, as mentioned for some other changes.